spec/observables/bindNodeCallback-spec.ts

```diff
@@ -8,6 +8,24 @@ const Observable = Rx.Observable;
 /** @test {bindNodeCallback} */
 describe('Observable.bindNodeCallback', () => {
   describe('when not scheduled', () => {
+    it('should emit undefined when callback is called without success arguments', () => {
+      function callback(cb) {
+        cb(null);
+      }
+
+      const boundCallback = Observable.bindNodeCallback(callback);
+      const results = [];
+
+      boundCallback()
+        .subscribe((x: any) => {
+          results.push(typeof x);
```

Comment:
I guess just habit of "everyone can set undefined value, so never trust it". All JS resources used to be very paranoid when I was learning. ;)